src/main/resources/hudson/plugins/synergy/SynergyFolderPublisher/config.jelly

```diff
@@ -1,11 +1,11 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-	<f:entry title="Copy folder only if the build is successful">
-		<f:checkbox name="synergyPublisher.publish" value="true" checked="${instance.isPublish()}"/>
+	<f:entry field="publish" title="Copy folder only if the build is successful">
+		<f:checkbox checked="${instance.isPublish()}" />

```

Comment:
BTW it is also common to move the `title` attribute from `f:entry` to `f:checkbox`, which gives nicer-looking layout in most cases (puts the label on the right of the checkbox and associates it in HTML as the label “for” the checkbox). Not important, but if you are visually checking the form anyway, you can consider doing it. For other controls like `f:textbox` the `title` must be on `f:entry`.